docs/models.yml

```diff
@@ -450,6 +457,12 @@ meeting:
     maxLength: 100
     default: OpenSlides
     restriction_mode: A
+  is_active:
+    type: relation
+    to: organization/active_meetings
+    restriction_mode: A
+    default: 1
```

Comment:
invalid attribute

Reply:
removed the default value, must be set in software to organization id